sigs.yml

```
@@ -0,0 +1,502 @@
+    chat:
+      - type: slack
+        name: '#otel-dotnet-auto-instr'
+        id: C01NR1YLSE7
+    invites: calendar-dotnet
+    tcSponsors: []
+    gcLiaison: []
+  - name: '.NET: SDK'
+    meeting: Every Tuesday alternating between 9:00 and 16:00 PT
+    notes:
+      type: gDoc
+      value: 1yjjD6aBcLxlRazYrawukDgrhZMObwHARJbB9glWdHj8
+    chat:
+      - type: slack
+        name: '#otel-dotnet'
+        id: C01N3BC2W7Q
+    invites: calendar-dotnet
+    tcSponsors: []
+    gcLiaison: []
+  - name: 'Erlang/Elixir: SDK'
+    meeting: Every other Thursday at 9:00 PT
+    notes:
+      type: gDoc
+      value: 1EbBiRjBc_cBf0T_B5OtNRPhbD4jdBrVYJAy8euCDrUI
+    chat:
+      - type: slack
+        name: '#otel-erlang-elixir'
+        id: C01N75YMZCN
+    invites: calendar-erlang
+    tcSponsors: []
+    gcLiaison: []
+  - name: 'GoLang: SDK'
+    meeting: Every Thursday alternating between 10:00 and 15:00 PDT
+    notes:
+      type: gDoc
+      value: 1E5e7Ld0NuU1iVvf-42tOBpu2VBBLYnh73GJuITGJTTU
+    chat:
+      - type: slack
+        name: '#otel-go'
+        id: C01NPAXACKT
+    invites: calendar-go
+    tcSponsors: []
+    gcLiaison: []
+  - name: 'GoLang: Automatic Instrumentation'
+    meeting: Every other week on Tuesday at 9:30 PT
+    notes:
+      type: gDoc
+      value: 1P6am_r_cxCX1HcpDQlznrTrTOvwN2whshL0f58lXSWI
+    chat:
+      - type: slack
+        name: '#otel-go-instrumentation'
+        id: C03S01YSAS0
+    invites: calendar-go
+    tcSponsors: []
+    gcLiaison: []
+  - name: 'Java: SDK + Automatic Instrumentation'
+    meeting: Every Thursday at 09:00 PT
+    notes:
+      type: gDoc
+      value: 1WK9h4p55p8ZjPkxO75-ApI9m0wfea6ENZmMoFRvXSCw
+    chat:
+      - type: slack
+        name: '#otel-java'
+        id: C014L2KCTE3
+    invites: calendar-java
+    tcSponsors: []
+    gcLiaison:
+    - name: Trask Stalnaker
+      github: trask
+  - name: 'JavaScript: SDK'
+    meeting: Every Wednesday at 09:00 PT
+    notes:
+      type: gDoc
+      value: 1tCyoQK49WVcE-x8oryZOTTToFm7sIeUhxFPm9g-qL1k
+    chat:
+      - type: slack
+        name: '#otel-js'
+        id: C01NL1GRPQR
+      - type: other
+        name: GitHub Discussions
+        link: https://github.com/open-telemetry/opentelemetry-js/discussions
+    invites: calendar-js
+    tcSponsors: []
+    gcLiaison:
+    - name: 'Daniel Dyla'
+      github: dyladan
+  - name: 'PHP: SDK'
+    meeting: Every Wednesday at 10:30 PT
+    notes:
+      type: gDoc
+      value: 1WLDZGLY24rk5fRudjdQAcx_u81ZQWCF3zxiNT-sz7DI
+    chat:
+      - type: slack
+        name: '#otel-php'
+        id: C01NFPCV44V
+    invites: calendar-php
+    tcSponsors: []
+    gcLiaison: []
+  - name: 'Python: SDK'
+    meeting: Every Thursday at 09:00 PT
+    notes:
+      type: gDoc
+      value: 1CIMGoIOZ-c3-igzbd6_Pnxx1SjAkjwqoYSUWxPY8XIs
+    chat:
+      - type: slack
+        name: '#otel-python'
+        id: C01PD4HUVBL
+    invites: calendar-python
+    tcSponsors: []
+    gcLiaison: []
```

```suggestion
    gcLiaison: []
```
@open-telemetry/governance-committee please assign yourself for being the liaison person for this SIG

---

```suggestion
    gcLiaison:
    - name: Daniel Gomez Blanco
      github: danielgblanco
```